### DISCUSSION TITLE
Candle effect for Philips HUE bulbs - nothing happens

### DISCUSSION THREAD

**Discussion by zimbo86** [Koenkk/zigbee2mqtt#20341] [Devices]
Hi, I´m running 1.30.4 with a Conbee II, Version 0x264a0700.
I can see the light effects (candle, fireworks) on my huebulbs, however when I activate them nothing happens.
When I pair the same hue bulbs natively to a hue bridge, light effects work as expected.
What am I doing wrong? Do I have to update my conbee II?
![image](https://github.com/Koenkk/zigbee2mqtt/assets/69269000/e2a547be-912e-4865-84b9-9dfdf95c028d)

**Comment by deviantintegral** [comment]
Since it works when you use the Hue app, I think you can rule out needing a firmware update.

However, I'm surprised - shouldn't you also have a colorloop and fireplace option? What bulb do you have exactly?

Also, while the effect list is visible, it does not work with groups for the candle and fireplace effects. So, make sure you're targeting an individual bulb and not a group.

**Comment by zimbo86** [comment]
I did not mean a fw update for the bulb but for the conbee stick.

I use E27 white ambiance bulbs, where just the candle effect should work. Fireplace etc need color bulbs. I got those versions of the bulb which all do nothing when enabling candle effect:
LTA001
LTW010
LWA001
LTA004

blink and other effects work.

**Reply by lux73** [reply]
i had to update my white ambiance Bulbs to latest Firmware to get "candlelight" working

I only have old Bulbs without Bluetooth - the newer ones with BT should support candle OotB

**Reply by zimbo86** [reply]
Can you provide your product code and version for these bulbs?

**Reply by lux73** [reply]
- LTW010
- LTW012
- LTW013

**Comment by zimbo86** [comment]
I´m sorry for the circumstances - i just noticed I never ever did an update of my z2m hue bulbs. 
In my "hue bridge" exmaple once the bulb was paired to the hue bridge the bulb was updated automatically the next day.
After doing an OTA update of a sample bulb in z2m (from 1.88.1 to 1.104.2) the candle effect works.
Never change a running system :D

**Comment by ooglyhLL** [comment]
> Also, while the effect list is visible, it does not work with groups for the candle and fireplace effects.
Does not _currently_ work, or _can't possibly_ work due to some protocol details that can't be fixed on z2m side?

**Reply by deviantintegral** [reply]
The bulbs themselves don't respond to the effect being sent to a group. I'd be surprised if Philips implemented it given I'm not sure they use groups themselves with Hue bridges for much.

<img width="557" alt="image" src="https://github.com/user-attachments/assets/a4b2612c-1ee0-4de9-9c78-3b5bc338f8d5">

zigbee2mqtt doesn't handle higher level automations today. For myself, I've created groups in Home Assistant, which then send commands to the individual bulbs and works fine. Whatever you're using to connect to zigbee2mqtt can likely to the same.

**Reply by ooglyhLL** [reply]
Too bad. I have a chandelier with those bulbs and it would have been a fun effect. Thanks for the hint regarding HA groups.
